packages/mdc-button/mdc-button.scss

```diff
@@ -17,65 +17,33 @@
 @import "@material/elevation/mixins";
 @import "@material/ripple/mixins";
 @import "@material/theme/mixins";
+@import "@material/theme/variables";
 @import "@material/typography/mixins";
 
 // postcss-bem-linter: define button
-
 .mdc-button {
   @include mdc-ripple-base;
   @include mdc-ripple-bg((pseudo: "::before"));
   @include mdc-ripple-fg((pseudo: "::after"));
+  @include mdc-typography(button);
   @include mdc-theme-prop(color, text-primary-on-light);
-  @include mdc-typography-base;
 
   display: inline-block;
   position: relative;
-  min-width: 64px;
+  min-width: 88px;
   height: 36px;
   padding: 0 16px;
   border: none;
-  border-radius: 2px;
+  border-radius: 4px;
```

Comment:
Is this some hint at a future change in guidelines, or a mistake that slipped through? Button style guidelines still currently specify 2dp corner radius.